utils.js

```diff
@@ -0,0 +1,76 @@
+function generateSummary(core, metrics) {
+  body = '### Lava scan results\n\n';
+
+  if (Object.keys(metrics.vulnerability_count).length === 0) {
+    return body + `No vulnerabilities found!! 🎉🎉🎉\n\n`;
+  }
+
+  body += `| Severity | Findings |\n`;
+  body += `|---|---:|\n`;
+
+  if (metrics.vulnerability_count["success"] > 0) {
+    body += `| :white_check_mark: success | ${metrics.scan_count["success"]} |\n`;
+  }
```

Comment:
Does `success` exist? Also, do you mean `vulnerability_count` instead of `scan_count`?

Reply:
Right, copilot things